server/src/exhibition/controller/exhibition.controller.ts

```diff
@@ -1,31 +1,51 @@
-import { Controller, Get } from '@nestjs/common';
+import { Controller, Get, ParseIntPipe, Query } from '@nestjs/common';
 import { ExhibitionService } from '../service/exhibition.service';
 import { Exhibition } from '../exhibition.entity';
 import {
-    ApiCreatedResponse,
     ApiOperation,
-    ApiProperty,
+    ApiProperty, ApiResponse,
     ApiTags,
 } from '@nestjs/swagger';
+import {
+    getExhibitionApiResponse, getExhibitionsSortedByDeadlineApiOperation, getExhibitionsSortedByInterestApiOperation,
+    getNewestExhibitionApiOperation,
+    getRandomExhibitionApiResponse,
+    getRandomExhibitionsAPiOperation,
+} from '../swagger';
 
-@Controller('/exhibitions')
 @ApiTags('전시회 컨트롤러')
-
+@Controller('/exhibitions')
 export class ExhibitionController {
     constructor(private exhibitionService: ExhibitionService) {}
 
     @Get('/random')
-    @ApiOperation({
-        summary: '전시회 랜덤 5개 조회 API',
-        description: '랜덤으로 전시회 5개를 조회한다.',
-    })
-    @ApiCreatedResponse({
-        description: '전시회 객체 5개 배열',
-        type: Exhibition,
-        isArray: true,
-    })
+    @ApiOperation(getRandomExhibitionsAPiOperation)
+    @ApiResponse(getRandomExhibitionApiResponse)
     @ApiProperty({})
     getRandomExhibitions(): Promise<Exhibition[]> {
         return this.exhibitionService.getRandomExhibitions();
     }
+
+    @Get('/newest')
+    @ApiOperation(getNewestExhibitionApiOperation)
+    @ApiResponse(getExhibitionApiResponse)
```

Comment:
```suggestion
    @ApiOperation(getNewestExhibitionApiOperation)
    @ApiResponse(getExhibitionApiResponse)
    @ApiQuery({name: "page", type: Number })
```
`@ApiQuery({name: "page", type: Number })` 도 있으면 좋을 듯!